IDF_template.txt

```diff
@@ -0,0 +1,65 @@
+MAGE-TAB Version
+Investigation Title
+Experiment Description
+
+Experimental Design
+Experimental Design Term Source REF
+Experimental Design Term Accession Number
+
+Experimental Factor Name
+Experimental Factor Type
+Experimental Factor Term Source REF
+Experimental Factor Term Accession Number
+
+Person Last Name
+Person First Name
+Person Mid Initials
+Person Email
+Person Phone
+Person Fax
+Person Affiliation
+Person Address
+Person Roles
+Person Roles Term Source REF
+Person Roles Term Accession Number
+
+Date of Experiment
+Public Release Date
+
+PubMed ID
+Publication DOI
+Publication Author List
+Publication Title
+Publication Status
+Publication Status Term Source REF
+Publication Status Term Accession Number
+
+Protocol Name
```

Comment:
suggested we give examples of the protocols used and link to the relative EFO terms

Reply:
Added the mandatory protocol types to the IDF template